modules/adtargetBidAdapter.md

```diff
@@ -0,0 +1,47 @@
+# Overview
+
+**Module Name**: Adtarget Bidder Adapter
+**Module Type**: Bidder Adapter
+**Maintainer**: kamil@adtarget.com.tr
+
```

Comment:
please add docs for the features you support, like `schain`, `userId`, etc.

for `userId` you can use `userIds: britepoolId, criteo, digitrust, id5Id, identityLink, liveIntentId, netId, parrableId, pubCommonId, unifiedId` since it looks like you listen to all available user ids.